lib/bunch/markdown.ex

```diff
@@ -0,0 +1,74 @@
+defmodule Bunch.Markdown do
```

Comment:
I would call the module `Bunch.String` as it can be useful for generating e.g. pure HTML or code either